### ISSUE TITLE
Animations Importer Possible?

### ISSUE THREAD

**Issue by DerekX** [Skarmory/mdl-importer#4]
Sorry for my previous complaint, had to get used to the setup. But now I wish this tool would also import animations. Can it be updated to do so?

**Comment by DerekX** [comment]
OR could AnimAll be used to "find" the animations of an imported MDL?

**Comment by Skarmory** [comment]
Yes it's unfortunate that it does not do the animations. It's been ages since I wrote this and I don't remember much about the Blender API. I will have a brief look at it to see how easy it will be to do BUT these days I'm pretty busy with other things in life so I will not make any promise that I will do it.

In regards to the second post, I don't know what AnimAll is, but since the script doesn't bother even reading the animation data at all I'm guessing no.
